boards/px4/fmu-v3/init/rc.board

```diff
@@ -47,6 +47,9 @@ hmc5883 -C -T -X start
 lis3mdl -X start
 ist8310 -C start
 
+# ICM20948 as external magnetometer on I2C (with Rotation 6 for Here GNSS)
+mpu9250 -X -M -R 6 start
```

Comment:
Side question, is the icm20948 actually mounted at yaw 270 in the Here GNSS or is this also accounting for rotations in the driver itself?